modules/libxcrypt/4.4.36/overlay/config/amd64-linux.h

```diff
@@ -0,0 +1,291 @@
+/* config.h.  Generated from config.h.in by configure.  */
```

Comment:
Is it possible to use https://registry.bazel.build/modules/cmake_configure_file to generate those headers?

Reply:
Unfortunately, `libxcrypt` does not have an `CMakeLists.txt` just Autotools `configure` script.

I can investigate how to generate the `config.h` from the `configure` script hermetically. Possibly using some of the `rules_foreign_cc` mechanisms.

Reply:
If it is acceptable, I'll swing back around to this. The configure script needs access to a load of the source code which is fine with `rules_foreign_cc` but a little challenging when attempting to do a native Bazel build. I have it passing on CI now by removing some of the modern detected APIs.

Reply:
OK, I'm fine with keeping it as it is, but in the long term we should still find a solution in case those headers get too big to review.